draft-irtf-cfrg-vdaf.md

```diff
@@ -3269,7 +3301,7 @@ def gen(IdpfPoplar, alpha, beta_inner, beta_leaf, binder, rand):
     if len(beta_inner) != IdpfPoplar.BITS - 1:
         raise ERR_INPUT # beta_inner vector is the wrong size
     if len(rand) != IdpfPoplar.RAND_SIZE:
-        raise ERR_INPUT # unexpected length for random coins
+        raise ERR_INPUT # unexpected length for random input
```

Comment:
We should replicate these changes in the `poc` directory so we don't accidentally undo them by copying in the other direction later.